src/data/openWeatherMapAPI.js

```diff
@@ -1,29 +1,20 @@
-import { OPEN_WEATHER_MAP_API_KEY } from '../secrets';
-
-export const cityCoordinates = {
-  London: {
-    lat: 51.5085,
-    lon: -0.1257,
-  },
-  Kyiv: {
-    lat: 50.4333,
-    lon: 30.5167,
-  },
-  Warsaw: {
-    lat: 52.2298,
-    lon: 21.0118,
-  },
-  Paris: {
-    lat: 48.8534,
-    lon: 2.3488,
-  },
-};
-
-export const allowedCities = Object.keys(cityCoordinates);
+/** @jsx createElement */
+/** @jsxFrag createFragment */
+import { createElement, createFragment } from '../framework/element';
```

Comment:
@nktssh why do we need these pragmas in data-related module?

Reply:
Obsolete (`<img/>` tag was here), will remove